utils/funcexp/func_concat_ws.cpp

```diff
@@ -94,10 +94,11 @@ string Func_concat_ws::getStrVal(Row& row,
 
 #else
     string str;
-
+    string tmp;
     for ( uint32_t i = 1 ; i < parm.size() ; i++)
     {
-        str += string(stringValue(parm[i], row, isNull).c_str());
+		string(stringValue(parm[i], row, isNull).c_str(), tmp);
```

Comment:
What was there before looks odd, turns a string to a c-string, then copies it.  In the new version since stringValue() returns nothing, can remove the string ctor around it.

Reply:
That is a good point, but since this is a merge from 1.1 to develop we should probably fix it there. I'll file a bug. In the mean time this isn't doing any harm, just a pain.